scripts/common.sh

```diff
@@ -200,8 +200,6 @@ function diff_values
 
     local value1="$1"
     local value2="$2"
-    shift
-    shift
```

Comment:
No, this two here are required, but you can replace them with `shift 2` since it is guaranteed that you will have at least two arguments (shift is ineffective if there are not enough arguments to be shifted).
Here they are needed because you forward the other parameters to the diff in line 204.

So, suppose that you call `diff_values "a" "b" --color` without the shift, the `"a"` and `"b"` would be passed to the diff as options, which is not what you want. In this case, you only want that `--color` to be forwarded.

Reply:
Ah yes! I missed that `"$@"`.